subt_ign/src/GameLogicPlugin.cc

```diff
@@ -777,6 +786,9 @@ void GameLogicPlugin::PostUpdate(
               std::chrono::duration_cast<std::chrono::steady_clock::duration>
               (std::chrono::seconds(s) + std::chrono::nanoseconds(ns));
 
+          // check if the robot has flipped
+          this->dataPtr->CheckRobotFlip(name, pose);
```

Comment:
For one `PostUpdate`, this runs once for every sensor in a robot. Can we limit it to run just once for a robot?

Reply:
I went ahead and moved the `CheckRobotFlip` call outside of the sensor portion in [1256865](https://github.com/osrf/subt/pull/519/commits/1256865b11f89d3a8393c5130f50261eaa93046a), so I believe it should only be called once per `PostUpdate` now.